test/index.test.js

```diff
@@ -42,66 +36,66 @@ it('returns error', () => {
 
 it('supports ES2016', () => {
   return getSize(fixture('es2016/index')).then(size => {
-    expect(size).toEqual(34)
+    expect(size).toEqual({ bundle: 47, gzip: 34 })
   })
 })
 
 it('support images', () => {
   return getSize(fixture('img/index')).then(size => {
-    expect(size).toEqual(53)
+    expect(size).toEqual({ bundle: 89, gzip: 53 })
   })
 })
 
 it('supports CSS', () => {
   return getSize(fixture('css/index')).then(size => {
-    expect(trim(size)).toEqual(2330)
+    expect(trim(size.gzip)).toEqual(2330)
   })
 })
 
 it('supports CSS modules', () => {
   return getSize(fixture('cssmodules/index')).then(size => {
-    expect(trim(size)).toEqual(2360)
+    expect(trim(size.gzip)).toEqual(2360)
   })
 })
 
 it('removes non-production code', () => {
   return getSize(fixture('multiple/production')).then(size => {
-    expect(size).toEqual(8)
+    expect(size).toEqual({ bundle: 14, gzip: 8 })
   })
 })
 
 it('ignores dependencies on request', () => {
   return getSize(fixture('peer/index'), { ignore: ['redux'] }).then(size => {
-    expect(size).toEqual(97)
+    expect(size).toEqual({ bundle: 222, gzip: 97 })
   })
 })
 
 it('disables webpack on request', () => {
   return getSize([
     fixture('bad/index'), fixture('es2016/index')
   ], { webpack: false }).then(size => {
-    expect(size).toEqual(122)
+    expect(size).toEqual({ bundle: 82, gzip: 122 })
```

Comment:
Found some weird thing. Gzipped result is bigger.

Reply:
Try to build project manually and check output.

Reply:
Yeah, I already tried. Seems like some dictionaries spends more than source.